pyfar/classes/coordinates.py

```diff
@@ -1667,9 +1667,9 @@ def find_nearest(self, find, k=1, distance_measure='euclidean',
                 index_multi = np.moveaxis(index, -1, 0)
                 index = np.empty((k), dtype=tuple)
                 for kk in range(k):
-                    index[kk] = tuple([index_multi[kk]], )
+                    index[kk] = tuple([index_multi[kk]] )
```

Comment:
Is there a reason to keep the whitespace before `)`?
Would think that violates another codestyle-rule.

Reply:
good point, I removed the space